.github/workflows/preDeploy.yml

```diff
@@ -138,9 +138,25 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
+  # Check if actor is member of Expensify organization
+  isOrganizationMember:
+    runs-on: ubuntu-latest
+
+    outputs:
+      isMember: ${{ fromJSON(steps.checkActor.outputs.isTeamMember) }}
+
+    steps:
+      - name: Check whether the actor is member of Expensify organization
+        id: checkActor
+        uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          username: ${{ github.actor }}
```

Comment:
I am using an action that has already been used in our codebase and their [documentation](https://github.com/tspascoal/get-user-teams-membership#usage) says that if we omit team and organization parameters, the default organization will be `github.repo_owner`. I believe that should be the Expensify organization we need, but just pointing this out for double-checking.

Reply:
Ah, thank you for pointing it out. I have submitted a PR to that repo to make that part clearer as I have missed it in the expectation they would list the name of the parameter there.

Looking further we can do two things by still using this action:
- look for this team which is in Expensify organization: https://github.com/orgs/Expensify/teams/expensify-expensify and should have all the full-time employees
- get the array of teams from the organization the actor is part of and if the array has a length > 0 then we can assume they are part of the organization, this might be better solutions so I will try to go with it